spec.html

```diff
@@ -30780,6 +30780,7 @@
         <emu-alg>
           1. Let _obj_ be ? ToObject(*this* value).
           1. Let _len_ be ? ToLength(? Get(_obj_, `"length"`)).
+          1. If _comparefn_ is not *undefined* and IsCallable(_comparefn_) is *false*, throw a *TypeError* exception.
```

Comment:
GitHub was hiding this part here. I'm also glad this is placed before the get length operation. +1 for this change.